src/ProfessionNPC.cpp

```diff
@@ -150,13 +205,28 @@ class NpcFreeProfessionsCreatureScript : public CreatureScript
                     player->learnSpell(51309);
                     break;
                 case SKILL_COOKING:
+                    player->learnSpell(2550);
+                    player->learnSpell(3102);
+                    player->learnSpell(3413);
+                    player->learnSpell(18260);
+                    player->learnSpell(33359);
                     player->learnSpell(51296);
                     break;
                 case SKILL_FIRST_AID:
+                    player->learnSpell(3273);
+                    player->learnSpell(3274);
+                    player->learnSpell(7924);
+                    player->learnSpell(10846);
+                    player->learnSpell(27028);
                     player->learnSpell(45542);
                     break;
                 case SKILL_FISHING:
-                    player->learnSpell(65293);
+                    player->learnSpell(7620);
+                    player->learnSpell(7731);
+                    player->learnSpell(7732);
+                    player->learnSpell(18248);
+                    player->learnSpell(33095);
+                    player->learnSpell(64484);
```

Comment:
minor: these ids could be replaced by enum so we know which spells correspond to the related ID